### ISSUE TITLE
Uncaught TypeError: panel.addClass is not a function when Swatches are disabled

### ISSUE THREAD

**Issue by VincentMarmiesse** [magento/magento2#20843]
### Preconditions (*)
1. Magento 2.3
2.PHP 7.2.14

### Steps to reproduce (*)
1. Disable Swatches modules
```
php bin/magento module:disable Magento_SwatchesLayeredNavigation Magento_SwatchesGraphQl Magento_Swatches
php bin/magento setup:upgrade
```
1. In admin, go to Stores > Attributes > Product
2. Edit a dropdown attribute (`country_of_manufacture` for example)

### Expected result (*)
1. You can update the dropdown options

### Actual result (*)
1. There is an error in console: `Uncaught TypeError: panel.addClass is not a function`

That's because when Swatches are disabled, the template `vendor/magento/module-catalog/view/adminhtml/templates/catalog/product/attribute/js.phtml` is loaded and has and error: it calls `addClass` function on a prototype Element. The same is done with `removeClass` function.

**Comment by magento-engcom-team** [comment]
Hi @VincentMarmiesse. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [x] Summary of the issue
- [x] Information on your environment
- [x] Steps to reproduce
- [x] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me 2.3-develop instance` - upcoming 2.3.x release

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@VincentMarmiesse do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [x] no

**Comment by VincentMarmiesse** [comment]
I can't reproduce it on vanilla Magento instance as I have to disable Swatches modules.

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [ ] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [ ] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-backlog-nazar
Thank you for verifying the issue! :+1: Your confirmation will help us to acknowledge and process this report.

**Comment by magento-engcom-team** [comment]
Hi @shikhamis11. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [x] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [x] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @shikhamis11 
 Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-98005`, `MAGETWO-98006` were created

**Issue Available**: @shikhamis11, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by Anantkprajapati** [comment]
Working on this #mm19in

**Comment by magento-engcom-team** [comment]
@Anantkprajapati thank you for joining. Please accept team invitation [here](https://github.com/orgs/magento/invitation) and self-assign the issue.

**Comment by magento-engcom-team** [comment]
Hi @Anantkprajapati. Thank you for working on this issue. 
Looks like this issue is already verified and confirmed. But if your want to validate it one more time, please, go though the following instruction:
- [x] 1. Add/Edit `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.

- [x] 2. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 3. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 4. If the issue is not relevant or is not reproducible any more, feel free to close it.

**Comment by Anantkprajapati** [comment]
Reproduced on 2.3.x http://prntscr.com/mfxxox

**Comment by Anantkprajapati** [comment]
Hello 

"There is an error in console: Uncaught TypeError: panel.addClass is not a function"
 This is fixed but after fix this getting another issue related to not load the value options 
So, Still working on this

**Comment by OhHeyAlan** [comment]
Reproduced on 2.3.1 when Magento_Swatches module is disabled. On the backend the "Save Attribute" button does not work.

`Uncaught TypeError: panel.removeClass is not a function`

`    at checkOptionsPanelVisibility (<anonymous>:43:19)
    at bindAttributeInputType (<anonymous>:50:5)
    at HTMLDocument.<anonymous> (<anonymous>:303:5)`

**Comment by itsazzad** [comment]
What is the solution?
We are also getting the same error.
Any workaround for now?

**Comment by OhHeyAlan** [comment]
Enable Swatches module then recompile and deploy.

**Comment by magento-engcom-team** [comment]
Hi @VincentMarmiesse. Thank you for your report. 
The issue has been fixed in magento/magento2#22560 by @markvds in 2.3-develop branch
Related commit(s):
 - [1abadb5ddf82505c2048dac9312a5d58f21e5992](https://github.com/magento/magento2/commit/1abadb5ddf82505c2048dac9312a5d58f21e5992)

The fix will be available with the upcoming 2.3.3 release.
